internal/engine/k8srollout/podmonitor.go

```diff
@@ -83,13 +87,22 @@ func (m *PodMonitor) OnChange(ctx context.Context, st store.RStore, _ store.Chan
 }
 
 func (m *PodMonitor) print(ctx context.Context, update podStatus) {
+	reusingPod := update.podStartTime.Before(update.updateFinishTime)
```

Comment:
this should probably use updateStartTime instead of updateFinishTime?

We've definitely seen cases where the pod finishes deploying before the Kubernetes request finishes, so it's entirely possible to have a new pod that starts before the update finishes.

Reply:
👍